### PR TITLE
adds package script for aarch64 Ubuntu 16.04

### PR DESCRIPTION
**Pull request by niranjan92**
https://github.com/Shippable/reqExec/issues/20

### CODE REVIEW

File: package_aarch64_Ubuntu16.04.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
+#!/bin/bash -e
```
Comment (shrivara) [inline package_aarch64_Ubuntu16.04.sh]: The file we're storing on S3 is called `reqExec-master-x86_64-Ubuntu.tar.gz` for x86_64. Maybe we can name package.sh similar: `package-aarch64-Ubuntu.sh`.

Comment (ric03uec) [reply package_aarch64_Ubuntu16.04.sh]: - for the file name, i think we shouldl follow the convention that we're using for naming the artifacts. so it should be `reqExec_<master>_x86_64_Ubuntu.tar.gz`
- package should be moved in a folder with same convention
```
package/aarch64/ubuntu14.04/package.sh
package/x86/ubuntu16.04/package.sh
```
- another thing i dont understand is how are binaries for both aarch and x86 being pushed to dist folder. they'll obviously override each other. im not sure i understand the entire artifact creation workflow end to end

Comment (shrivara) [reply package_aarch64_Ubuntu16.04.sh]: - We're following `reqExec-master-x86_64-Ubuntu.tar.gz` as the convention -- this way, things link `x86_64` will not be mixed up with `reqExec_master`
- We package each of this as separate binaries, so they will not overwrite each other. 

@ric03uec @niranjan92

**Comment by niranjan92** [comment]
will update pipeline scripts in next PR so exisitng pipeline works for x86_64

**Pull request merged**